src/intl/en/learn-quizzes.json

```
@@ -324,5 +324,41 @@
   "h005-c-label": "Eth1",
   "h005-c-explanation": "Eth1 was the original name given to the execution layer, not the consensus layer.",
   "h005-d-label": "Staking",
-  "h005-d-explanation": "Staking is depositing ETH into a smart contract to help secure the chain."
+  "h005-d-explanation": "Staking is depositing ETH into a smart contract to help secure the chain.",
+  "k001-prompt": "How is Ethereum scaling?",
```

Woridng feels weird

```suggestion
  "k001-prompt": "Which of the following is Ethereum using to scale?",
```

Don't love my suggestion either, but it think it's an improvement